packages/rest-crud/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@loopback/rest-crud",
+  "version": "0.1.0",
```

Comment:
Let's use `0.0.1` as the starting version as `lerna version` will increment it to `0.1.0` with the 1st commit such as `feat(rest-crud): add rest controller for crud`.